.github/workflows/run-acceptance-tests.yaml

```diff
@@ -85,7 +85,9 @@ jobs:
         run: |
           # Create GKE test cluster to install CRDs and use with the test operator.
           scripts/ci-cluster-create.sh
-
+          # Source the env variables created in the script above
+          cat ~/.envfile
```

Comment:
Is there anything sensitive in the env file that we don't want appearing in CI logs?

Reply:
Just the backend s3 bucket and the KMS key ARN. Neither are sensitive.